package.json

```diff
@@ -66,6 +66,8 @@
   },
   "license": "MIT",
   "dependencies": {
```

Comment:
Oh ok, I'm unsure as well but just wanted to point that out, I don't think it is harmful to keep it there or to remove it.